backend/app/services/adapters/task_kinds/queries.py

```
@@ -0,0 +1,977 @@
+        """
+        # Get task IDs
+        count_sql = text(
+            """
+            SELECT COUNT(*) FROM tasks
+            WHERE user_id = :user_id
+            AND kind = 'Task'
+            AND is_active = true
+            AND namespace != 'system'
+        """
+        )
+        total_result = db.execute(count_sql, {"user_id": user_id}).scalar()
```

Ruff (0.14.13)

Local variable `total_result` is assigned to but never used

Remove assignment to unused variable `total_result`

(F841)